### ISSUE TITLE
CIFAR100 pretrained models and results

### ISSUE THREAD

**Issue by curious-synapse** [stanford-futuredata/lit-code#2]
Could you please share your CIFAR100 pretrained models and/or results? (from here http://proceedings.mlr.press/v97/koratana19a/koratana19a.pdf) Thanks!

**Comment by ddkang** [comment]
Unfortunately we no longer have those models but they should be easy to train by modifying the CIFAR10 scripts.

**Comment by curious-synapse** [comment]
Thanks for the quick response! Do the same hyperparameters for CIFAR10 work on CIFAR100? Also, do you have the accuracy numbers (because the paper only has a plot) for ResNet and ResNeXt on CIFAR100 that I should be able to achieve?
